### PR TITLE
Fix: Patient Document issue

### PR DESCRIPTION
**Pull request by mupagasi**
This is a fix for the below screenshot error occurred on easyDoc when clicking on a patient in Patient Document module:

image

### CODE REVIEW

File: configuration/globalproperties/additional_clinical_gp.xml [hunk 1]
```diff
@@ -26,5 +26,72 @@
         ) order by v.date_started asc
     </value>
     </globalProperty>
+    <globalProperty>
```
Comment (rbuisson) [inline configuration/globalproperties/additional_clinical_gp.xml:29]: This file is probably there by mistake right?
Could you remove it from the PR?

**Review by rbuisson — CHANGES_REQUESTED** [review]
Thanks Jean Paul.
I am assuming the Waiting Room queue is added by mistake, as well as the Additional Visit Types zip file.
Could you remove them?

File: configuration/concepts/misc.csv [hunk 2]
```diff
@@ -0,0 +1,4 @@
+Uuid,Void/Retire,Same as mappings,Fully specified name:en,Short name:en,Description:en,Data class,Data type,Complex data handler,Answers,Members,Allow decimals,Units,Normal low,Normal high,Absolute low,Absolute high,_version:1,_order:1000
+81f330b5-3f10-11e4-adec-0800271c1b75,,,Document,Document,,Misc,Text,,,,,,,,,,,
```
Comment (rbuisson) [inline configuration/concepts/misc.csv:2]: Isn't this concept already created via inheriting the OpenMRS Config Haiti?
See here https://github.com/mekomsolutions/openmrs-config-haiti/blob/master/configuration/concepts/misc.csv#L34

**Review by rbuisson — CHANGES_REQUESTED** [review]
Seems like you are bringing concepts that are already brought by the OpenMRS Config Haiti.
Was this the real issue?

File: configuration/concepts/misc.csv (hunk 2 above)
Comment (mupagasi) [reply configuration/concepts/misc.csv:2]: Something is wrong here, when I do mvn clean install on this bahmni-distro-easydoc it doesn't bring this **Patient Document** concept.

What make me think that this error is from the **Patient Document** concept is that I have created them in openmrs administration and it works well.

File: configuration/globalproperties/additional_clinical_gp.xml (hunk 1 above)
Comment (mupagasi) [reply configuration/globalproperties/additional_clinical_gp.xml:29]: These commit are from an old PR, let me remove them

**Pull request closed without merging**